app/presenters/features.rb

```diff
@@ -1,60 +1,68 @@
 # frozen_string_literal: true
 
 class Features
-  # This is for toggling Wiki Ed-specific features that should not be active
-  # on any other dashboard instances.
   def self.wiki_ed?
-    return true if ENV['dashboard_url'] == 'dashboard.wikiedu.org'
-    return true if ENV['dashboard_url'] == 'dashboard-testing.wikiedu.org'
-    false
+    ENV['wiki_education'] == 'true'
   end
 
-  def self.default_course_type
-    ENV['default_course_type'] || 'ClassroomProgramCourse'
+  def self.enable_get_help_button?
+    ENV['wiki_education'] == 'true'
   end
 
-  def self.default_course_string_prefix
-    default_course_type.constantize.new.string_prefix
+  def self.wiki_trainings?
+    ENV['wiki_education'] != 'true'
+  end
+
+  def self.disable_help?
+    ENV['wiki_education'] != 'true'
+  end
+
+  def self.disable_help?
+    ENV['wiki_education'] != 'true'
   end
 
   def self.disable_onboarding?
-    ENV['disable_onboarding'] == 'true'
+    ENV['wiki_education'] != 'true'
+  end
+
+  def self.enable_language_switcher?
+    ENV['wiki_education'] != 'true'
   end
 
   def self.disable_help?
-    ENV['disable_help'] == 'true'
+    ENV['wiki_education'] != 'true'
   end
 
-  def self.disable_wiki_output?
-    ENV['disable_wiki_output'] == 'true'
+  def self.enable_account_requests?
+    ENV['wiki_education'] != 'true'
   end
 
-  def self.open_course_creation?
-    ENV['open_course_creation'] == 'true'
+  def self.disable_wiki_output?
```

Comment:
This should keep its old definition. Wiki output is currently enabled on both dashboard (and is usually disabled in dev environments).